modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -19,14 +19,20 @@ Validations include:
 
 * Targeting a machine config pool that belong to the machines to be tested
 * Enabling SCTP on the nodes
+* Enabling xt_u32 kernel module via machine config
 * Having the Performance Addon Operator installed
 * Having the SR-IOV Operator installed
 * Having the PTP Operator installed
 * Using OVN kubernetes as the SDN
 
-Latency tests, a part of the CNF-test container, also require the same validations. For more information about running a latency test, see the Running the latency tests section.
+Latency tests, a part of the CNF-test container, also require the same validations.
+For more information about running a latency test, see the Running the latency tests section.
 
-The tests need to perform an environment configuration every time they are executed. This involves items such as creating SRI-OV node policies, performance profiles, or PTP profiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SR-IOV node policy might result in the environment being temporarily unavailable until the configuration change is processed.
+The tests need to perform an environment configuration every time they are executed.
```

Comment:
Each of these sentences is on its own line, which doesn't seem right since it's not a bulleted list. We use soft wraps.

Reply:
I usually do that to make it easier for me to read. I'll start using soft wraps. :)